src/app.types.tsx

```diff
@@ -12,99 +12,63 @@ export const TAB_VALUES = [
 
-export interface AddCatalogueCategory {
-  name: string;
-  parent_id?: string | null;
-  is_leaf: boolean;
-  properties?: AddCatalogueCategoryProperty[];
-}
-
-export interface AddCatalogueCategoryWithPlacementIds
-  extends AddCatalogueCategory {
-  properties?: AddCatalogueCategoryPropertyWithPlacementIds[];
-}
-
-export interface EditCatalogueCategory {
-  name?: string;
-  id: string;
-  parent_id?: string | null;
-}
-
-export interface MoveToCatalogueCategory {
-  selectedCategories: CatalogueCategory[];
-  // Null if root
-  targetCategory: CatalogueCategory | null;
-}
-
-export interface CopyToCatalogueCategory {
-  selectedCategories: CatalogueCategory[];
-  // Null if root
-  targetCategory: CatalogueCategory | null;
-  // Existing known catalogue category names at the destination
-  // (for appending to the names to avoid duplication)
-  existingCategoryCodes: string[];
-}
-
-export interface CatalogueCategory {
-  id: string;
-  name: string;
-  parent_id: string | null;
-  code: string;
-  is_leaf: boolean;
-  properties?: CatalogueCategoryProperty[];
-  created_time: string;
-  modified_time: string;
-}
-
+// ------------------------------------ CATALOGUE CATEGORIES ------------------------------------
 export interface AllowedValuesList {
   type: 'list';
-  // eslint-disable-next-line @typescript-eslint/no-explicit-any
-  values: any[];
+  values: {
+    valueType: 'number' | 'string';
+    // eslint-disable-next-line @typescript-eslint/no-explicit-any
+    values: { av_placement_id: string; value: any }[];
+  };
 }
 export type AllowedValues = AllowedValuesList;
 
 export interface AddCatalogueCategoryProperty {
   name: string;
   type: string;
   unit_id?: string | null;
-  mandatory: boolean;
+  unit?: string | null;
+  mandatory: string;
   allowed_values?: AllowedValues | null;
-  default_value?: string | number | boolean;
 }
 
-export interface CatalogueCategoryPropertyMigration {
-  id?: string;
+export interface AddCatalogueCategoryWithPlacementIds {
   name: string;
-  type: string;
-  unit_id?: string | null;
-  mandatory: boolean;
-  allowed_values?: AllowedValues | null;
-  default_value?: string | number | boolean;
+  parent_id?: string | null;
+  is_leaf: string;
+  properties?: AddCatalogueCategoryPropertyWithPlacementIds[];
 }
 
-export interface AddPropertyMigration {
-  catalogueCategory: CatalogueCategory;
-  property: CatalogueCategoryPropertyMigration;
+export interface AddCatalogueCategoryPropertyWithPlacementIds
+  extends AddCatalogueCategoryProperty {
+  cip_placement_id: string; // Catalogue item properties (cip)
 }
 
-export interface EditPropertyMigration {
-  catalogueCategory: CatalogueCategory;
-  property: Partial<CatalogueCategoryPropertyMigration>;
+export interface AddPropertyMigration extends AddCatalogueCategoryProperty {
+  default_value: {
+    valueType: string;
+    // eslint-disable-next-line @typescript-eslint/no-explicit-any
+    value: { av_placement_id: string; value: any };
```

Comment:
I think this could do with a comment? Why does the form need an allowed value placement id here?